### ISSUE TITLE
Source code tarball name

### ISSUE THREAD

**Issue by bdubbs** [htacg/tidy-html5#506]
Can you please change the name of the source tarball from 5.4.0.tar.gz to tidy-html5-5.4.0.tar.gz.  If downloading via wget, the stem of the filename is not present without a -O option.  this is not conducive to lists of wget urls.

  -- Bruce Dubbs
    linuxfromscratch.org

**Comment by balthisar** [comment]
@bdubbs, those are generated automatically by Github, I'm afraid. Without re-writing all of our tags, do you have a suggestion on how to accomplish this?

**Comment by bdubbs** [comment]
Yes it is a problem with github not recognizing that not all users fetch source files with a browser.

Most projects have a link of some kind.  See for instance https://github.com/logrotate/logrotate/releases/

**Comment by balthisar** [comment]
Got it. Done.

**Comment by geoffmcl** [comment]
@balthisar thank you for adding these - `tidy-html5-5.4.0.tar.gz` and `tidy-html5-5.4.0.zip` - [here](https://github.com/htacg/tidy-html5/releases/tag/5.4.0), within a few hours...

However I note the `zip` source contains unix line endings! 

That is what I would expect in the `tar.gz`, but it is my general idea that a `zip` would have Windows, CRLF, line ending... or is this an incorrect assumption on my part?

It is not too important since **most** Windows editors, and compilers, handle both easily... the exception of course is the very dumb `notepad.exe` that is still shipped with Windows... and MSVC will refuse an xml solution file with unix line endings the  last time I checked, but there are none of these in the source...

Some unzip utilities, like WinZip, also include an auto line ending translation, but this can lead to a big mess in a binary file, and should usually be left **off**... but again these are no binary files in this source...

It should be further noted, that provided a Windows user has setup the **git config** correctly, when they clone the source repo **all** files will have CRLF, and git will warn on commit if you add a unix line ending source... and the MSVC IDE editor will also warn about such a file, and offer to normalize it... but of course, on a git push, only the default repo unix line endings will be uploaded...

I could replace the zip, if desired... but as stated not too important...

**Comment by balthisar** [comment]
I simply downloaded the files that Github provided. For some reason when downloaded, they has the names that @bdubbs wanted automatically, so I simply dragged them into the releases.

I guess I'm saying, Github managed the line endings that way for some reason.

FWIW, the ZIP format is also the default archive format on Mac OS X, and it uses Unix line endings, so I guess the only assumption I would make is that the unarchiver should change line endings itself!
